docs/gettingstarted/structuring.md

```diff
@@ -1,54 +1,83 @@
+`@Mod` is the annotation indicating that the class is a *Mod entry point* for Forge Mod Loader. It contains various bits of Metadata that the mod will use as well as modifiers that tell Forge how to handle your mod.
 
-You can find an example mod in the [Forge src download](http://files.minecraftforge.net/).
+Now, the `@Mod` annotation can take quite a few arguments, but the only three *required* ones are `modid`, `name`, and `version`.
+* `modid` is the unique identifier and *internal* name of the mod, and takes a `String`. Usually, this is the same name as the mod, but in all lowercase. It's used internally by Forge and by other Mod Developers that wish to find your mod through Forge (for instance, to use your APIs). 
+ * **ONCE YOU PICK THIS IDENTIFIER, YOU SHOULD NOT CHANGE IT DURING THE LIFE OF THE MOD. BAD THINGS HAPPEN IF YOU DO AND OTHER MODS DEPEND ON YOUR MOD.**
+* `name` is the *external* name of the mod, and takes a `String`.
+* `version` is the version of the mod, and takes a `String`. There is no set standard for how you version things, but it's recommended that you read the [versioning](https://mcforge.readthedocs.io/en/latest/conventions/versioning/) article.
 
-Keeping Your Code Clean Using Sub-packages
-------------------------------------------
+Simple, right? `@Mod` has a wide range of other things you can throw at it, but those will be covered in the articles that use them.
 
-Rather than clutter up a single class and package with everything, it is recommended you break your mod into subpackages.
+What is `@Instance`?
+-------------------
+`@Instance` is actually a child annotation of `@Mod`. It basically just gives Forge and other Mod Developers a way to instance your mod if they need to.
 
-A common subpackage strategy has packages for `common` and `client` code, which is code that can be run on server/client and client, respectively. Inside the `common` package would go things like Items, Blocks, and Tile Entities (which can each in turn be another subpackage). Things like GUIs and Renderers would go inside the `client` package.
+What is `@EventHandler`?
+-----------------------
+`@EventHandler` is an annotation used to mark whatever method it's attached to as a handler for a *Forge Lifecycle Event*. There are two main categories of Lifecycle Events: **Standard** and **Server**. **Standard Lifecycle Events** are always fired when the client and server first starts up, and **Server Lifecycle Events** are only fired when the server, either internal or dedicated, starts up. There are some specialized events as well, but those will be covered in a different atricle.
 
-!!! note
+For a simple mod, *FMLPreInitializationEvent*, *FMLInitializationEvent*, and *FMLPostInitializationEvent* should be more than enough to set up everything you need.
```

Comment:
Add a link to the [loading stages](https://mcforge.readthedocs.io/en/latest/conventions/loadstages/) here.